ariadne/contrib/tracing/utils.py

```diff
@@ -0,0 +1,35 @@
+from graphql import GraphQLResolveInfo, ResponsePath
+
+from ...resolvers import is_default_resolver
+
+
+def format_path(path: ResponsePath):
+    elements = []
+    while path:
+        elements.append(path.key)
+        path = path.prev
+    return elements[::-1]
+
+
+def should_trace(info: GraphQLResolveInfo):
+    if info.field_name not in info.parent_type.fields:
+        return False
+
+    resolver = info.parent_type.fields[info.field_name].resolve
+    if (
```

Comment:
We could just return expression value (inversed).

Reply:
Nice catch!